oshmem/mca/sshmem/ucx/sshmem_ucx.h

```diff
@@ -49,10 +49,19 @@ sshmem_ucx_shadow_allocator_t *sshmem_ucx_shadow_create(unsigned count);
 void sshmem_ucx_shadow_destroy(sshmem_ucx_shadow_allocator_t *allocator);
 int sshmem_ucx_shadow_alloc(sshmem_ucx_shadow_allocator_t *allocator,
                             unsigned count, unsigned *index);
+
+/* reallocate existing allocated buffer. if possible - used inplace
```

Comment:
minor: better start with uppercase after the dot

Reply:
fixed